packages/web/src/components/gcds-file-uploader/gcds-file-uploader.tsx

```diff
@@ -333,6 +333,47 @@ export class GcdsFileUploader {
     this.internals.setFormValue(formData);
   };
 
+  /*
+   * Handle drop event on file uploader
+   */
+  private handleDrop(e) {
+    e.preventDefault();
+
+    const droppedFiles = e.dataTransfer.files;
+
+    if (droppedFiles.length > 0) {
+      const dt = new DataTransfer();
+      for (const file of droppedFiles) {
+        if (this.isFileAccepted(file)) {
+          dt.items.add(file);
+        }
+      }
+
+      if (dt.files.length > 0) {
+        this.shadowElement.files = dt.files;
+        this.files = dt.files;
+      }
+    }
```

Comment:
Can we add an error message to give the user some feedback if something goes wrong?

Reply:
An error message has been added if the `isFileAccepted` function returns `false`